cdc/model/sink.go

```diff
@@ -295,9 +295,19 @@ type RowChangedEvent struct {
 
 	RowID int64 `json:"row-id" msg:"-"` // Deprecated. It is empty when the RowID comes from clustered index table.
 
-	Table     *TableName         `json:"table" msg:"table"`
-	ColInfos  []rowcodec.ColInfo `json:"column-infos" msg:"-"`
-	TableInfo *TableInfo         `json:"-" msg:"-"`
+	// Table contains the table name and table ID.
+	// NOTICE: We store the physical ID here, not the logical ID.
+	Table    *TableName         `json:"table" msg:"table"`
+	ColInfos []rowcodec.ColInfo `json:"column-infos" msg:"-"`
+	// NOTICE: We probably store the logical ID inside TableInfo's TableName,
+	// not the physical ID.
+	// For partitioned table, there are two kinds of ID:
+	// 1. TIDB_PARTITION_ID is the physical ID of the partition.
+	// 2. TIDB_TABLE_ID is the logical ID of the table.
+	// In general, we always use the physical ID to represent a table, but we
+	// record the logical ID from the DDL event(job.BinlogInfo.TableInfo).
+	// So be careful when using the TableInfo.
+	TableInfo *TableInfo `json:"-" msg:"-"`
```

Comment:
Please also add comments at structure definition of `TableName` and `TableInfo`.